packages/core/src/temporal/Temporal.js

```diff
@@ -148,9 +152,13 @@ export class Temporal extends TemporalAccessor {
      * @throws DateTimeException - if the unit cannot be subtracted
      * @throws ArithmeticException - if numeric overflow occurs
      */
-    // eslint-disable-next-line no-unused-vars
-    minusAmountUnit(amountToSubtract, unit) {
-        abstractMethodFail('minusAmountUnit');
+    _minusUnit(amountToSubtract, unit) {
+        requireNonNull(amountToSubtract, 'amountToSubtract');
+        requireNonNull(unit, 'unit');
+        requireInstance(unit, TemporalUnit, 'unit');
+        return (amountToSubtract === MIN_SAFE_INTEGER
+            ? this._plusUnit(MAX_SAFE_INTEGER, unit)._plusUnit(1, unit)
```

Comment:
This code was taken from `DefaultInterfaceTemporal` which in turn was taken from the ThreeTen, but I think is incorrect in JavaScript.

In Java this equality holds:

```
-(Long.MIN_VALUE) == Long.MAX_VALUE + 1
```

But in JS the next is true instead:

```
-(Number.MIN_SAFE_INTEGER) == Number.MAX_SAFE_INTEGER
```

So adding 1 more unit is wrong here.
Besides, in Java a `long` (or any integer) just go back to its minimum value in an overflow situation, and that is the expected behavior of the original code I think, but in JS the `number` will just increase beyond `MAX_SAFE_INTEGER` (until it reaches `Number.MAX_VALUE` or `Infinity`):

```
Number.MAX_SAFE_INTEGER + 1 > Number.MAX_SAFE_INTEGER
```

The code should at least be modified to remove the `._plusUnit(1, unit)` part (and in any other places where that's relevant). If the code should try to emulate the integer-modular semantic of Java is something I'm not sure.

There is a new commented test that right now fails for this code.